modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -38,6 +39,123 @@
+
+=== Properly remove file versions from the trashbin
+
+Previously, restoring or removing a file from inside a folder that was deleted (so the folder and the contents were in the trashbin) didn't remove the versions of the file. Those versions were left in both the DB and the FS, taking space and degrading the performance. This is now being handled properly, so no additional resource is consumed due to the versions being left stranded. https://github.com/owncloud/core/pull/40291[#40291]
+
+=== "available for" select in the mount point configuration now shows the displaynames
+
+The "available for" select of the mount configuration of external storages was using the group id. This wasn't a problem because for local groups the group id matches the group displayname, and for ldap groups the group id was the "cn" attribute. Due to recent changes, the ldap group will now use the "objectuid" attribute (or a similar attribute) as group id by default. This was causing the "available for" select to show that "objectuid", so identifying the right group was problematic. Now, the "available for" select will show the group displayname, which for ldap is the "cn" attribute by default. Note that this happens on new installations. There is an automatic migration in place, so for upgrades, the "cn" attribute will be set as groupname in order to keep the old behavior. https://github.com/owncloud/core/pull/40412[#40412]
+
+=== Skip public links when updating permissions of share's children
+
+Updates to permissions of a share were wrongly propagated to public links children. This has now been fixed and public links are being skipped. https://github.com/owncloud/core/pull/40420[#40420]
+
+=== Store checksums only if the whole stream has been read
+
+Previously, range downloads (or downloads requesting a specific byte range) would store a checksum, if needed, based only on the requested range. This caused problems because the checksum is expected to be for the whole file. Now those range downloads won't store a checksum because only a part of the file has been read, so the checksum would be incomplete. Some additional cases have been taken into account, mostly based on actions that could happen on the data stream, but they shouldn't happen normally. https://github.com/owncloud/core/pull/40513[#40513]
+
+=== Fix the dav:cleanup-chunks command to work with a configured folder
+
+The ownCloud's FS was initialized partially to prevent contacting the LDAP server if it was configured. This was causing problems because the upload folder where the chunks were expected was a mountpoint, and due to the partial FS initialization such mountpoint was missing, so we were checking a different folder (the default one). Now, the ownCloud's FS will be fully initialized instead, so the mountpoint will be present and we check the right location. https://github.com/owncloud/core/pull/40571[#40571]
```

Comment:
```suggestion
ownCloud's file system was initialized partially to prevent contacting the LDAP server if it was configured. This was causing problems because the upload folder where the chunks were expected was a mount point, and due to the partial file system initialization such a mount point was missing, so we were checking a different folder (the default one). Now, ownCloud's file system will be fully initialized instead, so that the mount point will be present and we check the right location. https://github.com/owncloud/core/pull/40571[#40571]
```